.travis.yml

```diff
@@ -100,7 +110,7 @@ _helm_deploy: &helm_deploy
 
     # failed pods logs
     failed_pod_logs() {
-      pod_status | grep False | awk '{print $1}' | while read pod; do echo -e '\e[1;31m' $pod '\e[0m' && kubectl logs $pod --namespace $namespace; done
+      pod_status | grep False | grep -v Succeeded | awk '{print $1}' | while read pod; do echo -e '\e[1;31m' $pod '\e[0m' && kubectl logs $pod --namespace $namespace; done
```

Comment:
The reindexing is performed by a Kubernetes Job. This means that the Pod terminates with status Succeded when it finishes its job. The previous check was considering only running pods.